tests/test_ci_attestation_retry.sh

```
@@ -0,0 +1,57 @@
+    exit 1
+  fi
+
+  echo "PASS: $(basename "$file") retries required remote daemon asset attestation"
+}
+
+check_attestation_retry \
+  "$ROOT_DIR/.github/workflows/nightly.yml" \
+  "Attest remote daemon nightly assets" \
+  "attest-remote-daemon-nightly-assets" \
+  "Retry remote daemon nightly asset attestation" \
+  'remote-daemon-assets/cmuxd-remote-manifest-${{ env.NIGHTLY_BUILD }}.json'
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)